### PR TITLE
Ensure gitpod user has the correct GID

### PR DESCRIPTION
**Pull request by axonasif**
## Context:

https://github.com/renovatebot/renovatebot.github.io/pull/244

Try here: https://gitpod.io/#https://github.com/renovatebot/renovatebot.github.io/pull/326

### CODE REVIEW

File: .gitpod/Dockerfile [hunk 1]
```diff
@@ -20,7 +20,7 @@ SHELL ["/bin/bash" , "-c"]
 # Set up buildpack
 COPY --from=buildpack /usr/local/bin/ /usr/local/bin/
 COPY --from=buildpack /usr/local/buildpack/ /usr/local/buildpack/
-RUN install-buildpack
+RUN install-buildpack && usermod -g ${USER_ID} ${USER_NAME}
```
Comment (viceice) [inline .gitpod/Dockerfile:23]: so the primary group needs to be gitpod?

Comment (axonasif) [reply .gitpod/Dockerfile:23]: No, only the GID needs to be 33333, for some reason it wasn't being set to that as intended.

Comment (viceice) [reply .gitpod/Dockerfile:23]: ok, the primary gid was set to `0`, which was intentional to allow write access to specific folders and support Openstack.

Comment (axonasif) [reply .gitpod/Dockerfile:23]: > ok, the primary gid was set to 0, which was intentional to allow write access to specific folders and support Openstack.

If I understand your setup correctly, `DOCKERD_ARGS` might be helpful. You can refer to [this link](https://www.gitpod.io/docs/configure/projects/environment-variables#user-specific-environment-variables) for more information.

> So the primary group (gid) needs to be 33333 to work?

I think so.

**Pull request merged**